.circleci/images/plume-buildenv/cargo_config

```diff
@@ -1,3 +1,3 @@
 [target.x86_64-unknown-linux-gnu]
 # link dead code for coverage, attempt to reduce linking memory usage to not get killed
-rustflags = ["-Clink-dead-code", "-Clink-args=-Xlinker --no-keep-memory -Xlinker --reduce-memory-overheads"]
+rustflags = ["-Clink-args=-Xlinker --no-keep-memory -Xlinker --reduce-memory-overheads"]
```

Comment:
Don't we need to link dead code anymore?

Reply:
It's somewhere it this line now https://github.com/Plume-org/Plume/pull/558/files#diff-1d37e48f9ceff6d8030570cd36286a61R74